### ISSUE TITLE
딥러닝 모델 예측 참고자료입니다.

### ISSUE THREAD

**Issue by JSY8869** [JSY8869/Us-su#123]
predict함수의 경우 입력받은 값을 모델의 각 노드를 거쳐 가중치를 계산하여 유사도를 비교한 값을 출력합니다.
그렇기 때문에 배열에 매우 이상한 값이 나오게 되는데 이 값은 모델의 신뢰도를 나타냅니다. 이 중에서 가장 신뢰도가 높은 값에 해당하는 label 값이 예측에 해당하는 답이 됩니다.
https://www.tensorflow.org/tutorials/keras/classification?hl=ko
